geopandas/geodataframe.py

```diff
@@ -51,7 +51,7 @@ class GeoDataFrame(GeoPandasBase, DataFrame):
         column on GeoDataFrame.
     """
 
-    _metadata = ["crs", "_geometry_column_name"]
+    _metadata = ["_crs", "_geometry_column_name"]
```

Comment:
Was this needed?

Reply:
Yes

Reply:
Sorry for the late reply here, but do you remember *why* ?

Reply:
I don't remember the exact error I got at the moment. But, it caused some error in pandas. (I wasn't able to reply here on my phone for some reason ...)